README.AKASA.md

```diff
@@ -19,10 +21,14 @@ Akasa maintains a small fork of the [Playwright Python](https://github.com/micro
   * `cp ../playwright/utils/build/output/*.zip driver/`
   * Change driver_version in `setup.py` to match above version.
   * Comment out `use_scm_version` in setup.py and add `version` to match above version.
+  * Create a file called playwright/_repo_version.py with code `version=` with the value being the version used above for setup.py 
+  * Generate the sync and async apis
```

Comment:
How is this done?

Reply:
the dev would manually create a file under playwright where it's body would be version='x' for example. if you're asking about whether this is automated it's done manually.